renderer/containers/ModalStack.js

```diff
@@ -75,9 +117,7 @@ function ModalStack(props) {
         /* eslint-disable react/display-name */
         (styles => (
           <Container style={styles}>
-            <Modal onClose={() => closeModal(modal.id)}>
-              <ModalContent closeModal={closeModal} type={modal.type} />
-            </Modal>
+            <ModalContent closeModal={() => closeModal()} type={modal.type} />
```

Comment:
can be just `closeModal={closeModal}` now

Reply:
Updated.